src/services/company.js

```diff
@@ -189,6 +190,26 @@ class CompanyService {
         }
     }
 
+
+    /**
+     * Edit company details by its ID and returns it
+     * @param {*} companyId ID of the company
+     * @param {*} companyDetails company details to be updated
+     */
+    async editCompanyDetails(companyId, companyDetails) {
+        try {
+            const company = await Company.findOneAndUpdate(
+                { _id: companyId },
+                companyDetails,
+                { new: true }
+            );
+
+            return company;
+        } catch (err) {
+            console.error(err);
+            throw err;
+        }
+    }
```

Comment:
We have a `changeAttributes` function so this behavior is duplicated: either use that or change the function's name.